hack/update-godep-licenses.sh

```diff
@@ -147,18 +147,16 @@ declare -Ag CONTENT
 echo "================================================================================"
 echo "= Kubernetes licensed under: ="
 echo
-cat ${LICENSE_ROOT}/LICENSE
+cat "${LICENSE_ROOT}/LICENSE"
 echo
-echo "= LICENSE $(cat ${LICENSE_ROOT}/LICENSE | md5sum | awk '{print $1}')"
+echo "= LICENSE $(md5sum < "${LICENSE_ROOT}/LICENSE" | awk '{print $1}')"
```

Comment:
FWIW I find this form harder to read, but if that's what shellcheck wants, so be it.

Reply:
Gotta optimize for those `pipe()` calls :D

Reply:
I agree - I sort of hate this...

Reply:
can we push back on shellcheck?

Reply:
Sure, we can. I'm only fixing a subset of findings anyway, because there are also false positives. In the end, if we want to have a "clean shellcheck", we can annotate the issues to make shellcheck ignore them, or we can even leave out issue classes with shellcheck configuration options.

Anyway, I'll remove this particular "fix" and push the patches again...